fuel/config_parser.py

```diff
@@ -96,6 +96,24 @@ def extra_downloader_converter(value):
     return value
 
 
+def multiple_paths_parser(value):
+    """Parses data_path argument.
+
+    Parameters
+    ----------
+    value : str
+        a string of data paths separated by  ":".
+
+    Returns
+    -------
+    value : list
+        a list of strings indicating each data paths.
+
+    """
+    value = value.split(os.path.pathsep)
```

Comment:
Yes that solves the problem. 

So the "value" passed to multiple_paths_parser() could be a list, which can only happen when it is already an output of multiple_paths_parser(). Does that mean, config.add_config('data_path', ...) is called even if after parsing command line parameters?